Content.Server/Mobs/CritMobActionsSystem.cs

```diff
@@ -58,6 +69,7 @@ private void OnLastWords(EntityUid uid, MobStateActionsComponent component, Crit
         _quickDialog.OpenDialog(actor.PlayerSession, Loc.GetString("action-name-crit-last-words"), "",
             (string lastWords) =>
             {
+                // Intentionally does not check for muteness
```

Comment:
Please check for `CanSpeak()` here as well.

Reply:
The entire start of this PR was the premise that this oversight should be embraced as a feature. Mirrorcult and Deltanedas were both supportive of the idea on the linked issue (19019)

Reply:
For context https://github.com/space-wizards/space-station-14/issues/19019#issuecomment-1676025825